libpod/runtime.go

```diff
@@ -362,6 +362,35 @@ func SetXdgRuntimeDir() error {
 	if err := os.Setenv("XDG_RUNTIME_DIR", runtimeDir); err != nil {
 		return errors.Wrapf(err, "cannot set XDG_RUNTIME_DIR")
 	}
+
+	// Setup XDG_CONFIG_HOME
+	cfgHomeDir := os.Getenv("XDG_RUNTIME_DIR")
+
+	cfgHomeDirLinger, err := rootless.EnableLinger()
```

Comment:
Thought: is this necessary? personally, I already have a systemd service for services that should survive a logout. It also fails for me locally (for reasons that I didn't bother investigating).